Server/src/main/java/Database/TheAuPair/Services/UserService.java

```diff
@@ -69,4 +71,24 @@ public User login(String email, String password)
     }
     return nr;
   }
+
+  public List<User> getApplicants()
+  {
+    List<User> u =  ur.findAllByType(1);
+    return u;
+  }
+
+  public void resolveApplication(String id, String resolution)
+  {
+    if (resolution.equals("approve"))
+    {
+      User u =  ur.findUsingId(id);
+      u.setRegistered(true);
+      ur.save(u);
+    }
+    else if (resolution.equals("decline"))
```

Comment:
Please add an 'else'  in case it isn't decline or approve. Just for future-proofing and will help to find errors more easily if we maybe pass Approve instread of approve by accident

Reply:
The service that calls this takes a boolean that decides what to pass through these are the only options that will be present